--- ../cryptoauthlib/lib/atca_iface.h	2018-01-16 02:01:00.000000000 +0100
+++ atca_iface.h	2018-02-28 16:16:51.345542100 +0100
@@ -72,19 +72,19 @@

     union                           // each instance of an iface cfg defines a single type of interface
     {
-        struct ATCAI2C
+        struct // ATCAI2C
         {
             uint8_t  slave_address; // 8-bit slave address
             uint8_t  bus;           // logical i2c bus number, 0-based - HAL will map this to a pin pair for SDA SCL
             uint32_t baud;          // typically 400000
         } atcai2c;

-        struct ATCASWI
+        struct // ATCASWI
         {
             uint8_t bus;        // logical SWI bus - HAL will map this to a pin	or uart port
         } atcaswi;

-        struct ATCAUART
+        struct // ATCAUART
         {
             int      port;      // logic port number
             uint32_t baud;      // typically 115200
@@ -93,7 +93,7 @@
             uint8_t  stopbits;  // 0,1,2
         } atcauart;

-        struct ATCAHID
+        struct // ATCAHID
         {
             int      idx;           // HID enumeration index
             uint32_t vid;           // Vendor ID of kit (0x03EB for CK101)
@@ -102,7 +102,7 @@
             uint8_t  guid[16];      // The GUID for this HID device
         } atcahid;

-        struct ATCACUSTOM
+        struct // ATCACUSTOM
         {
             ATCA_STATUS (*halinit)(void *hal, void *cfg);
             ATCA_STATUS (*halpostinit)(void *iface);
